docs/source/config/backup_restore.md

```diff
@@ -13,7 +13,8 @@ High level listing of all available config at this level follows:
 
 ```yaml
 backup:
-  url: ...
+  url/connection: ...
```

Comment:
i wouldn't write "url/connection" here, seems confusing

Reply:
maybe: `url:  # alternatively, 'connection:'`  if you want to show both ways here